desktop/src/main.js

```diff
@@ -12,7 +13,6 @@ const CONFIG = require('./config');
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 // eslint-disable-next-line
-let win;
```

Comment:
Why did you remove this line?

Reply:
Because `win` is unused in the entirety of the file lol

Reply:
I mean I can leave it there but the variable is unused.

Reply:
I know it's unused - the comment above says to do that though? Idk